changelog/add-jcb-payemnt-method

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
There is a small typo in the name of this file: `payemnt` => `payment`

Reply:
Thank you, resolved!